File: Selenium/Activities/Activity12.java
package Activity;

import java.time.Duration;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.support.ui.ExpectedConditions;
import org.openqa.selenium.support.ui.WebDriverWait;

public class Activity12 {
	
	public static void main(String[] args) {
		
		WebDriver driver = new FirefoxDriver();
		WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(30));
		
		driver.get("https://training-support.net/webelements/dynamic-content");
		System.out.println("Title of the page: " + driver.getTitle());
		
		driver.findElement(By.id("genButton")).click();
		
		if (wait.until(ExpectedConditions.textToBePresentInElementLocated(By.id("word"), "release"))) {
			
			System.out.println("Word Found: " + driver.findElement(By.id("word")).getText());
		
		}
		driver.quit();
	}

}

File: Selenium/Activities/Activity12.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.support.wait import WebDriverWait
from selenium.webdriver.support import expected_conditions

with webdriver.Firefox() as driver:
    wait = WebDriverWait(driver, timeout=30)
    driver.get("https://training-support.net/webelements/dynamic-content")
    print("Title of the page is: ", driver.title)

    driver.find_element(By.ID, "genButton").click()

    if wait.until(expected_conditions.text_to_be_present_in_element((By.ID, "word"), "release")):
        print("Word found: ", driver.find_element(By.ID, "word").text)